source/localizable/routing/defining-your-routes.md

```diff
@@ -28,6 +28,7 @@ Router.map(function() {
   this.route('favorites', { path: '/favs' });
 });
 ```
+If your route name has multiple works then your route should follow convention and be hyphenated, not underscored. Routes with underscores in them will not be rendered.
```

Comment:
Can you tweak an existing example, or add a new one? Also, I think it suffices to say that Ember expects the names to be dasherized.